extract/extract/extractors/amazon_de.py

```diff
@@ -238,6 +238,7 @@ def parse_image_urls(images: BeautifulSoup) -> list[str]:
             str(image["src"])
             for image in images.find_all("img")
             if not image["src"].endswith(".gif")
+            and "IMAGERENDERINGjpg" not in image["src"]
```

Comment:
Are we sure we don't want them in our db at all?
I thought this would be handled in the ecosia-exporter, or maybe I missed something in the meeting?
I think this way Ecosia will still get those products, but this time with no url at all, which I thought would be an issue as well. @BigDatalex what do you think?

Reply:
Ah, good find!